cmd/dcratomicswap/main.go

```diff
@@ -403,9 +400,9 @@ func normalizeAddress(addr string, defaultPort string) (hostport string, err err
 
 func walletPort(params *chaincfg.Params) string {
 	switch params {
-	case &chaincfg.MainNetParams:
+	case chaincfg.MainNetParams():
```

Comment:
check the net id here (params.Net) against the wire constants instead.  this old version was comparing the addresses of the global variables, but the functions in the more recent chaincfg will point to different objects

Reply:
done